### ISSUE TITLE
Dad Joke

### ISSUE THREAD

**Issue by JaydenPahukula** [ColoradoSchoolOfMines/acm-discord-bot#14]
- Whenever someone says something like "I am hungry", the bot will respond with "Hi hungry, I am [acm_bot]!", or something similar.
- This would need to scan every new message in every channel, as opposed to working with a slash command. I'm not sure how to do do this, but it is probably possible.

**Comment by killerontheradio** [comment]
Hi! I am very new to oss. Can I work on this as a first project?

**Comment by JaydenPahukula** [comment]
@pizzanapoletana yeah sure! Are you in the acm discord? We will talk more about the project and how to get started in the #discord-bot channel so stay tuned

**Comment by killerontheradio** [comment]
@JaydenPahukula hey there. I am extremely sorry for the late. I was tied up in several university projects and couldn't get back to you. Really, really sorry about it. I am not in the acm discord. And I am not even a student of Colorado School of Mines. If that's fine, I would love to join the discord and work with you. Again, I am extremely sorry.

**Comment by JaydenPahukula** [comment]
@pizzanapoletana Hey its all good. This project is mostly for fun, so don't worry if you get too busy or something, there's no hard contribution requirements. [Here](https://discord.gg/hwUspgXKm2) is the link to the Mines ACM discord, we'll communicate in the discord-bot channel but feel free to hang out and stuff!

**Comment by killerontheradio** [comment]
@JaydenPahukula hey there. Discord says the link is invalid. :(

**Comment by JaydenPahukula** [comment]
@pizzanapoletana oops try [this](https://discord.com/invite/E2FnnrfdQ8)

**Comment by killerontheradio** [comment]
Thank you! That worked and I am in the discord now. :)
